packages/console/src/assets/docs/guides/web-go/README.mdx

```diff
@@ -247,25 +235,22 @@ func main() {
 }`}
 </Code>
 
-Now, when your user visit `http://localhost:8080/sign-in`, the user will be redirected to the Logto sign-in page.
+Now, when your user visit `http://localhost:3000/sign-in`, the user will be redirected to the Logto sign-in page.
```

Comment:
can you rebase master and use `defaultBaseUrl` exported from `_redirect-uris-web.mdx`?